src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+  width: 1275px
+  height: 670px;
+  background: url(${retrotoolScreenshot}) no-repeat;
+  filter: drop-shadow(0 0 1px rgba(0,0,0,.5));
+`;
+
+const RetroSteps = styled(Text).attrs({
+  as: "h3",
+  textStyle: "landingHowtoStep"
+})`
+  ::before {
+    content: "";
+    position: absolute;
+    margin-top: 12px;
+    margin-left: -40px;
+    width: 30px;
+    height: 1px;
+    background: ${themeGet("colors.violet")};
+  }
+`;
+
+const Landing: React.FC<Props> = () => (
+  <>
+    <LandingGlobalStyle />
+    <Box bg="white" margin="auto">
+      <Flex
+        as="header"
+        justifyContent="space-between"
+        alignItems="center"
+        p={6}
+      >
+        <Logo />
+        <Flex>
+          <Text mr={3}>
+            <a href="https://twitter.com/marciobarrios" tabIndex={3}>
+              Contact us
+            </a>
+          </Text>
+          ·
+          <Text ml={3}>
+            <a href="https://github.com/retro-tool/" tabIndex={4}>
+              Github
+            </a>
+          </Text>
+        </Flex>
+      </Flex>
+      <Flex as="main" flexDirection="column">
+        <Section
+          textAlign={["left", null, "center"]}
+          maxWidth="100ch"
+          margin="auto"
+          py={[24, null, null, 120, 150]}
+          px={6}
+        >
+          <Text as="h1" fontSize={[7, 8, 9, 10]} lineHeight={7}>
+            Retro tool,{" "}
+            <Box position="relative" as="span">
+              simple and effective{" "}
+              <Underline display={["none", null, null, "block"]} />
+            </Box>{" "}
+            retrospectives for your team
+          </Text>
+          <Box mt={6} mb={[8, null, 10]} mx="auto" maxWidth="60ch">
+            <Text fontSize={[5, null, 6]} color="grey">
+              Collaborate with your remote team to improve what you do, guided
+              by a simple workflow. And for free!
+            </Text>
+          </Box>
+          <Flex mb={2} justifyContent="center" textAlign="center">
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="/new"
+              variant="landingPrimary"
+              fontSize={[3, null, 4]}
+              tabIndex={1}
+              px={[4, null, 7]}
+            >
+              Create a new retro
+            </Button>
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="#how-it-works"
+              variant="landingSecondary"
+              ml={5}
+              tabIndex={2}
+              fontSize={[3, null, 4]}
+              px={[4, null, 7]}
+            >
+              See how it works
+            </Button>
+          </Flex>
+          <Text
+            fontSize={[3, null, 4]}
+            color="grey"
+            mt={7}
+            mb={[4, null, null, 0]}
+          >
+            Already{" "}
+            <Text fontSize="inherit" color="dark" as="span" fontWeight="bold">
+              1.259 retros
```

Comment:
is this ought to be dynamic? or not on this iteration?

Reply:
The query is not ready yet right?

Reply:
the PR is open, but yeah it's not merged yet, waiting for a review. We can do it in the next iteration.

There's currently 672 retros on the production database btw.